server/server_test.go

```diff
@@ -70,6 +72,12 @@ func checkResponseCode(t *testing.T, expected, actual int) {
 	}
 }
 
+func checkHeaders(t *testing.T, expected, actual string) {
+	if expected != actual {
```

Comment:
Why not to use assert.Equal instead of this function?

Reply:
Because it's basically copy of `checkResponseCode`, and in this case there is not so much differences, because headers is one line strings, so it doesn't matter if we check it through `!=` or through `assert`

Reply:
assert shows you old version, new and the diff of objects you're comparing